modules/manage/pages/manage-xdcr/create-xdcr-replication.adoc

```diff
@@ -54,12 +55,15 @@ Left-click on the *XDCR* tab, in the left-hand navigation menu.
 [#left_click_on_xdcr_tab]
 image::manage-xdcr/left-click-on-xdcr-tab.png[,90,align=left]
 +
-This displays the *XDCR Replications* screen, the lower part of the main panel of which is entitled *Outgoing Replications*:
+This displays the *XDCR Replications* screen, which lists the following:
+
+** *Remote Clusters*: Lists the remote clusters that have been registered as references on the current cluster.
+** *Outgoing Replications*: Lists the details of replications that are being created from the current cluster to a remote cluster.
```

Comment:
Lists the details of replications that have been created on the current cluster.